src/ray/protobuf/experimental/instance_manager.proto

```diff
@@ -15,109 +15,141 @@
 syntax = "proto3";
 option cc_enable_arenas = true;
 
-package ray.rpc.autoscaler;
+package ray.rpc.autoscaler.im;
 
-enum RayNodeKind {
-  UNKNOWN = 0;
-  HEAD = 1;
-  WORKER = 2;
+// A status code to be used for operations in the Instance Manager
+// and other components.
+enum StatusCode {
+  // Success operations.
+  OK = 0;
```

Comment:
I remember as best practice, 0 should be UNSPECIFIED?

Reply:
oh, i remember it now.